samples/Extensions/Extensions.csproj

```diff
@@ -23,6 +22,10 @@
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
   </ItemGroup>
+   <ItemGroup>
+     <!-- Sample uses project reference, but actual apps should use 'PackageReference' -->
+     <ProjectReference Include="../../extensions/Worker.Extensions.ServiceBus/src/Worker.Extensions.ServiceBus.csproj" />
```

Comment:
Ah I see this project has a bunch of different samples in it.

@fabiocav , @liliankasem  - what do you two think of using p2p refs for this project? Package references makes it difficult to add new in-development features to the samples. Or is that intentional and this should only demonstrate already release features?

Reply:
I believe this was intentional to only demo released features; FunctionApp is the sample project that uses project references for testing. I typically only update samples for extensions once they've released and in their own PR.

(I'm not opposed to switching this, just sharing what I'm aware of)

Reply:
Given that explanation, @aishwaryabh do you have any issue saving the sample change till this is released?

Reply:
@liliankasem is correct, but I think this is indeed causing unnecessary churn and may also lead to stale samples. No need to block, but let's discuss changes do this.

Reply:
Sure no worries I can remove these samples for now then and create another PR once the worker packages have been released